src/qs8-gemm/scalar.c.in

```diff
@@ -3,9 +3,9 @@
 // This source code is licensed under the BSD-style license found in the
 // LICENSE file in the root directory of this source tree.
 
-$assert DATATYPE in ["QD8", "QC4"] or REQUANTIZATION in ["FP32", "RNDNU"]
+$assert DATATYPE in ["QD8", "QC4", "QB4_F16", "QB4_F32"] or REQUANTIZATION in ["FP32", "RNDNU"]
```

Comment:
Nice, doesn't seem like we are touching the existing generated kernel (since there aren't any changes checked in!) :)

Reply:
Yeah, I did run generate-qs8-gemm and it comes back clean, so there should be no changes to existing kernels.